src/updateManager.ts

```diff
@@ -43,9 +43,6 @@ export class UpdateManager {
       if (rows.length) {
         await trx(this.tableName).where('id', this.rowId).update({
           mv_locked: true,
-          // We set this to newer than isoTS, while we do the migration
-          // If the migration fails, we can reset the lock
-          mv_ts: ExportHelper.utcTS(),
```

Comment:
The reason for this is so that in multi container architecture that you don't accidentally have multiple containers trying to run imports or sync at the same time. That is why it is in a transaction.

Reply:
True, guess I just wanted to be double sure :D